articles/azure-signalr/includes/signalr-quickstart-run-web-application.md

```diff
@@ -10,22 +10,27 @@ ms.custom: include file
 ---
 ## Run the web application
 
-1. There is a sample single page web application hosted in GitHub for your convenience. Open your browser to [https://azure-samples.github.io/signalr-service-quickstart-serverless-chat/demo/chat-v2/](https://azure-samples.github.io/signalr-service-quickstart-serverless-chat/demo/chat-v2/).
+1. To convenient your client testing, open your browser to our sample single page web application [https://azure-samples.github.io/signalr-service-quickstart-serverless-chat/demo/chat-v2/](https://azure-samples.github.io/signalr-service-quickstart-serverless-chat/demo/chat-v2/). 
 
     > [!NOTE]
-    > The source of the HTML file is located at [/docs/demo/chat-v2/index.html](https://github.com/Azure-Samples/signalr-service-quickstart-serverless-chat/blob/master/docs/demo/chat-v2/index.html).
+    > The source of the HTML file is located at [/docs/demo/chat-v2/index.html](https://github.com/Azure-Samples/signalr-service-quickstart-serverless-chat/blob/master/docs/demo/chat-v2/index.html). And if you'd like to host the HTML yourself, please start a local HTTP server such as [http-server](https://www.npmjs.com/package/http-server) in the */docs/demo/chat-v2* directory. Ensure the origin is added to the `CORS` setting in *local.settings.json* similar to the sample.
+    ```js
+    "Host": {
+      "LocalHttpPort": 7071,
+      "CORS": "http://localhost:8080,https://azure-samples.github.io",
+      "CORSCredentials": true
+    }
+    
+    ```
 
-1. When prompted for the function app base URL, enter `http://localhost:7071`.
+2. When prompted for the function app base URL, enter `http://localhost:7071`.
```

Comment:
It's not necessary to update the bullets with the correct number. Leaving them as `1.` makes it easier to add steps or reorder steps.